app/views/lessons/show.html.erb

```diff
@@ -79,14 +79,67 @@
           <% end %>
         <% end %>
       </div>
+      <div class="card-footer">
+        <% if @reaction && current_user? %>
+          <div class='col text-right'>
+            <h5 class='my-1 pr-2'>You said this note made you feel <%= @reaction.reaction_name %>.
+              <%= sanitize(emojify(":#{@reaction.reaction_name}:", "selected-emoji")) %>
+            </h5>
+            <div class='row justify-content-end mt-4'>
+              <p class='my-auto col-auto'>
+                <%= link_to 'Remove',
+                  course_lesson_reaction_path(
+                    course_id: @course.slug,
+                    lesson_id: @lesson.slug,
+                    reaction_name: @reaction
+                  ),
+                  method: :delete %>
+                my reaction or update it:
+              </p>
+              <div>
+                <% @possible_reactions.select { |r| r != @reaction.reaction_name }.each do |r| %>
+                  <%= render "reaction", reaction: r %>
+                <% end %>
+              </div>
+            </div>
+          </div>
+        <% elsif current_user? %>
+          <div class='row float-right align-middle'>
+            <h6 class='my-1 pr-4'>How does this note make you feel?</h6>
+            <div class="col">
+              <% @possible_reactions.each do |r| %>
+                <%= render "reaction", reaction: r %>
+              <% end %>
+            </div>
+          </div>
+      <% else %>
+        <h6 class='my-2'>To get these notes emailed directly to you
+          <%=
+            link_to(
+              "create an subscription.",
+              course_subscription_path(@course),
+            )
+          %>
+        </h6>
+        <h6 class='my-2'>Or you can
+          <%=
+            link_to(
+              "sign in",
+              auth_sign_in_path(course_id: @course.to_param),
```

Comment:
Signing in from the lesson page doesn't redirect you back to the lesson page (it takes you to the course page for that lesson).

I don't think this is show-stopping, so can always address this in a separate PR, after this gets in.

Reply:
Ahh yes I did this on purpose though so they could see their previous reactions? Let's do in a separate PR

Reply:
👍